feature/policy_forwarding/GUE/Outer_IPv4/decapsulation/otg_tests/Inner_IP/README.md

```diff
@@ -0,0 +1,337 @@
+
+### PF-1.4.4: GUE Decapsulation of inner IPv6 traffic using non-default GUE UDP port or port-range
+-  Push DUT configuration.
+
+Traffic: 
+-  Generate IPv6 GUE-encapsulated traffic from ATE Port 1 with destinations matching DECAP-DST and configured non-default GUE UDP port or port-range. 
+-  Inner IPv6 destination should match IPV6-DST1.
+-  Inner-packet traffic-class should be set to 128.
+-  Inner-packet TTL value should be set to 64.
+
+Verification:
+-  Decapsulated IPv6 traffic is received on ATE Port 2.
+-  No packet loss.
+-  Inner-packet traffic-class should be preserved.
+-  Inner-packet TTL value should be decremented by 1 to 63.
+-  PF counters reflect decapsulated packets.
+
+### PF-1.4.5: GUE Decapsulation of inner IPv4 traffic using default GUE UDP port 6080 and GUE checksum ALL-ZERO
+-  Push DUT configuration.
+
+Traffic: 
+-  Generate GUE-encapsulated traffic from ATE Port 1 with destinations matching DECAP-DST and default GUE UDP port 6080.
+-  GUE checksum is set to ALL-ZERO.
+-  Inner IPv4 destination should match IPV4-DST1.
+-  Inner-packet DSCP value should be set to 32.
+-  Inner-packet TTL value should be set to 64.
+
+Verification: 
+-  Decapsulated IPv4 traffic is received on ATE Port 2.
+-  No packet loss.
+-  Inner-packet DSCP should be preserved.
+-  Inner-packet TTL value should be decremented by 1 to 63.
+-  PF counters reflect decapsulated packets.
+
+### PF-1.4.6: GUE Decapsulation of inner IPv4 traffic using non-default GUE UDP port or port-range and GUE checksum ALL-ZERO
+-  Push DUT configuration.
+
+Traffic: 
+-  Generate GUE-encapsulated traffic from ATE Port 1 with destinations matching DECAP-DST and configured non-default GUE UDP port or port-range.
+-  GUE checksum is set to ALL-ZERO.
+-  Inner IPv4 destination should match IPV4-DST1.
+-  Inner-packet DSCP value should be set to 32.
+-  Inner-packet TTL value should be set to 64.
+
+Verification: 
+-  Decapsulated IPv4 traffic is received on ATE Port 2.
+-  No packet loss.
+-  Inner-packet DSCP should be preserved.
+-  Inner-packet TTL value should be decremented by 1 to 63.
+-  PF counters reflect decapsulated packets.
+
+### PF-1.4.7: GUE Decapsulation of inner IPv6 traffic using default GUE UDP port 6615 and GUE checksum ALL-ZERO
+-  Push DUT configuration.
+
+Traffic: 
+-  Generate IPv6 GUE-encapsulated traffic from ATE Port 1 with destinations matching DECAP-DST and GUE UDP port 6615.
+-  GUE checksum is set to ALL-ZERO.
+-  Inner IPv6 destination should match IPV6-DST1.
+-  Inner-packet traffic-class should be set to 128.
+-  Inner-packet TTL value should be set to 64.
+
+Verification:
+-  Decapsulated IPv6 traffic is received on ATE Port 2.
+-  No packet loss.
+-  Inner-packet traffic-class should be preserved.
+-  Inner-packet TTL value should be decremented by 1 to 63.
+-  PF counters reflect decapsulated packets.
+
+### PF-1.4.8: GUE Decapsulation of inner IPv6 traffic using non-default GUE UDP port or port-range and GUE checksum ALL-ZERO
```

Comment:
Where did this port range specification come from?

I'd suggest that given that all these tests depend on some definition of a configuration schema that is as yet undefined, someone drafts the required configuration ASAP.

Reply:
The port range requirement is in b-265024996.